src/main/java/com/toptrumps/cli/CommandLineView.java

```diff
@@ -201,6 +201,12 @@ public void showRoundResult(RoundOutcome outcome, List<Card> winningCards) {
         typePrint(20, outcomeMessage);
     }
 
+    public void selectNextRound(Scanner scanner){
```

Comment:
@rjs68 It's a good idea to keep things like "asking for input" in the ```Controller```. The view is only the UI component -- everything you need to print out on the screen. The ```Controller``` orchestrates the request for input, computation of game logic (model), and presentation of data (view).